datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -271,6 +271,19 @@ def create_scraper_configuration(self, instance=None):
 
         config['telemetry'] = is_affirmative(instance.get('telemetry', default_instance.get('telemetry', False)))
 
+        # The metric name services use to indicate build information
+        config['metadata_metric_name'] = instance.get(
+            'metadata_metric_name', default_instance.get('metadata_metric_name', 'version')
+        )
+
+        # Map of metadata key names to label names
+        config['_default_metadata_label_map'] = {'version': 'version'}
```

Comment:
I'm not sure I get what we're trying to achieve there?

There's no garantee to have a `version` metrics in the payload 🤔 (as you had to add it in the fixtures)

Reply:
Right, that's why you can override it..